src/mgmt/srv.c

```diff
@@ -258,7 +258,7 @@ process_startms_request(Drpc__Call *drpc_req, Drpc__Response *drpc_resp)
 			       false /* !bootstrap */, NULL /* uuid */,
 			       NULL /* addr */);
 	if (rc == -DER_ALREADY) {
-		D_DEBUG(DB_MGMT, "MS already started\n");
+		D_INFO("MS already started\n");
```

Comment:
Inclined to leave this one as debug, myself. Just because the request is satisfied either way.